core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -61,11 +61,26 @@ def start_navigator(self) -> None:
         ## Can be changed back to a simple command after https://github.com/ArduPilot/ardupilot/issues/17572
         ## gets fixed.
         # pylint: disable=consider-using-with
+        #
+        # The mapping of serial ports works as in the following table:
+        #
+        # |    ArduSub   |       Navigator         |
+        # | -C = Serial1 | Serial1 => /dev/ttyS0   |
```

Comment:
@willian, -C = SERIAL1_*?, use the parameter indices

Reply:
Yes, that is how I did it